<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttDialog</class>
 <widget class="QDialog" name="AttDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="CBoxItems">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>211</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelItem">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Item</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="TableAtts">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>431</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PButtonAddAtt">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/img/Plus.png</normaloff>:/img/Plus.png</iconset>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PButtonRemoveAtt">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/img/Minus.png</normaloff>:/img/Minus.png</iconset>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>176</width>
     <height>31</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="upPushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/img/Up.png</normaloff>:/img/Up.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="downPushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>280</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/img/Down.png</normaloff>:/img/Down.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images/images.qrc"/>
 </resources>
 <connections/>
</ui>
